File: commutil/src/main/java/com/comm/util/room/AppDatabase.java
package com.comm.util.room;

import androidx.room.Database;
import androidx.room.RoomDatabase;

@Database(entities = {BleEntity.class},version=1)
public abstract class AppDatabase extends RoomDatabase {

    public abstract BleDao bleDao();

//   public static final Migration MIGRATION_1_2 = new Migration(1, 2) {
//
//        @Override
//        public void migrate(@NonNull SupportSQLiteDatabase database) {
//            database.execSQL("ALTER TABLE BleEntity "
//                    + " ADD COLUMN bleCode INTEGER");
//        }
//    };

}

File: androidUtil/src/main/java/com/android/util/room/AppDatabase.kt
package com.android.util.room
//
//import androidx.room.Database
//import androidx.room.RoomDatabase
//import androidx.room.migration.Migration
//import androidx.sqlite.db.SupportSQLiteDatabase
//import com.android.util.room.box.BleDao
//import com.android.util.room.box.BleEntity
//import com.android.util.room.officer.UserDao
//
//@Database(entities = [BleEntity::class], version = 1)
//abstract class AppDatabase : RoomDatabase() {
//    abstract fun bleDao(): BleDao?
//
//    abstract fun userDao():UserDao
//
//
//    companion object {
//        val MIGRATION_1_2: Migration = object : Migration(1, 2) {
//            override fun migrate(database: SupportSQLiteDatabase) {
//                database.execSQL(
//                    "ALTER TABLE BleEntity "
//                            + " ADD COLUMN bleCode INTEGER"
//                )
//            }
//        }
//    }
//}